scripts/get_version.mjs

```diff
@@ -43,7 +43,7 @@ export async function getVersion(platform) {
     case "windows":
       return "1.7.0";
```

Comment:
The current [latest Windows version is still 1.6.0](https://github.com/Jigsaw-Code/outline-releases/blob/master/client/latest.yml), we did not release 1.7.0 yet, so no need to change here.